.gitignore

```diff
@@ -129,8 +129,8 @@ config.log
 
 # End of https://www.gitignore.io/api/clion,cmake,scons
 
-# uncomment when https://github.com/godotengine/godot/issues/71521 fix is in release.
```

Comment:
We can delete the .godot repo now safely and exclude it from version control, as https://github.com/godotengine/godot/issues/69511 is fixed already.